DC/Lab10/EOSC350Lab.cls

```diff
@@ -0,0 +1,153 @@
+%==============================
```

Comment:
yes I did not moved the .tex and the .pdf but I forgot the .cls (which is the just the layout). I remove it